Create CONTENT_TEMPLATE.md (#36)
WriteSpeakCode/writespeakcode.com: .github/ISSUE_TEMPLATE/CONTENT_TEMPLATE.md

---
name: Add content to the website

> about: For adding content - can be completed entirely in CMS

Reviewer: non-blocking suggestion

```suggestion
about: For adding content - can be completed entirely in CMS, or via a github pull request
```

You can update content with a pull request, yea?

Author: yes ✨

---

### Content type
<!--
Is this an issue to add or update a speaker? sponsor? copy? metadata? etc
-->

#### Content Details

> <!-- Add all the information for the content and upload any images here if applicable -->

Reviewer: non-blocking suggestion

```suggestion
<!-- Add information about the content and upload any images here, if applicable -->
```
